frontend/components/Sidebar.tsx

```diff
@@ -0,0 +1,192 @@
+import { Button } from 'antd';
+import { ReactElement, useMemo, useState } from 'react';
+import {
+  MenuOutlined,
+  UserOutlined,
+  BookOutlined,
+  QuestionCircleOutlined,
+  BarChartOutlined,
+} from '@ant-design/icons';
+import styled from 'styled-components';
+import slideStyle from './slide.module.scss';
+
+const MenuHeading = styled.h1`
+  color: #800080;
+`;
+
+const MenuElements = styled.div`
+  margin-left: 10px;
+  :hover {
+    color: #800080;
+  }
+`;
+
+const SidebarDiv = styled.div`
+  position: absolute;
+  x: 0;
+  y: 0;
+  width: 30vw;
```

Comment:
When I resize my screen to be a mobile size view, the SidebarDiv is too smal
<img width="506" alt="Screen Shot 2021-11-07 at 5 17 37 PM" src="https://user-images.githubusercontent.com/17476012/140663856-b7d96802-d87f-4c4f-a50f-b2f6da74af83.png">
l